ts/parts-3d/Area.ts

```diff
@@ -0,0 +1,362 @@
+/* *
+ *
+ *  (c) 2010-2020 Torstein Honsi
+ *
+ *  Area 3D series.
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import type StackingAxis from '../parts/StackingAxis';
+import type SVGPath from '../parts/SVGPath';
+import H from '../parts/Globals.js';
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+        interface Area3dPointOptions extends AreaPointOptions {
+            z?: number;
+        }
+        interface Area3dPointLike {
+            x?: number;
+            y?: number;
+            z?: number;
+            plotX?: number;
+            plotY?: number;
+            plotZ?: number;
+            doCurve?: boolean;
+        }
+        interface Area3dSeriesOptions extends AreaSeriesOptions {
+        }
+        interface SeriesTypesDictionary {
+            area3d: typeof Area3dSeries;
+        }
+        class Area3dPoint extends AreaPoint {
+            public options: Area3dPointOptions;
+            public series: Area3dSeries;
+        }
+        class Area3dSeries extends AreaSeries {
+            public data: Array<Area3dPoint>;
+            public options: Area3dSeriesOptions;
+            public pointClass: typeof Area3dPoint;
+            public points: Array<Area3dPoint>;
+        }
+    }
+}
+
+import Point from '../parts/Point.js';
+import U from '../parts/Utilities.js';
+const {
+    seriesType,
+    pick
+} = U;
+
+var Series = H.Series;
+
+
+/**
+ * @private
+ * @class
+ * @name Highcharts.seriesTypes.area3d
+ *
+ * @augments Highcharts.Series
+ */
+seriesType(
+    'area3d',
+    'area',
+    /**
+     * A 3D area plot uses x, y and z coordinates to display values for three
+     * variables for a set of data.
+     *
+     * @sample {highcharts} highcharts/3d/area/
+     *         Simple 3D area
+     *
+     * @extends      plotOptions.area
+     * @excluding    dragDrop, cluster
+     * @product      highcharts
+     * @requires     highcharts-3d
+     * @optionparent plotOptions.area3d
+     */
+    {
+        tooltip: {
+            pointFormat: 'x: <b>{point.x}</b><br/>y: <b>{point.y}</b><br/>z: <b>{point.z}</b><br/>'
+        }
+
+    // Series class
+    }, {
+        getGraphPath: function (
```

Comment:
It seems to be a copy-paste from area with minor changes. Maybe it's time to split `getGraphPath` into minor methods?